go.mod

```
@@ -1,10 +1,10 @@
 module github.com/getgauge-contrib/json-report
 
-go 1.24
+go 1.24.5
```

```suggestion
go 1.24
```

No need to change this, GHA always runs with the latest version available.